pkgs/applications/version-management/git-and-tools/bitbucket-server-cli/default.nix

```diff
@@ -0,0 +1,19 @@
+{ lib, bundlerEnv, ruby }:
+
+bundlerEnv rec {
+  name = "bitbucket-server-cli-${version}";
+
+  version = (import gemset).atlassian-stash.version;
+  inherit ruby;
+  gemfile = ./Gemfile;
+  lockfile = ./Gemfile.lock;
+  gemset = ./gemset.nix;
```

Comment:
Yeah I don't think that would be possible in this context. The gemset would be null by and the passthru gems not available yet.